File: Practicas/GPIO/gpio.c
/** @brief: Este programa configura el BCM_GPIO 17 usando la libreria wiringPi
 * El GPIO 17 corresponde con el pin 0 en wiring Pi
 * Se debe compilar con: gcc gpio.c -o gpio -lwiringpi
 */

#include<stdio.h>
#include<unistd.h>
#include<wiringPi.h>

int main( )
{
//Se inicializa la libreria wiring Pi
	wiringPiSetup();
//Se configura el GPIO 17 como salida
	pinMode( 0, OUTPUT );

//Se escribe un valor digital al GPIO
	while( 1 )
	{
		digitalWrite( 0, HIGH );
	//	sleep(5);
		digitalWrite( 0, LOW );
	//	sleep(5);
	}
	return 0;
}

File: Practicas/GPIO/gpio.py
#Este programa configura el BCM_GPIO 17 usando phyton
#El BCM_GPIO 17 corresponde con la terminal 11 del conector
#de 40 terminales de la raspberry

import RPi.GPIO as GPIO
import time

#Se inicializa los GPIOS
GPIO.setmode(GPIO.BOARD)
#Se configura el BCM_GPIO 17 como salida usando la terminal 11
GPIO.setup(11, GPIO.OUT)

#Se escribe un valor digital al BCM_GPIO 17
while(1):
	GPIO.output(11,GPIO.HIGH)
#	time.sleep(1)
	GPIO.output(11,GPIO.LOW)
#	time.sleep(1)
